apps/metatube/overlays/nishir-tailnet/kustomization.yaml

```
@@ -10,3 +12,16 @@ labels:
       app.kubernetes.io/name: metatube
       app.kubernetes.io/part-of: nishir-media
       app.kubernetes.io/version: 1.4.0
+patches:
+  - path: patch-netpol.yaml
+    target:
+      name: metatube
+      version: v1
+      kind: NetworkPolicy
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)